src/rules/function-url-data-uris/index.js

```diff
@@ -0,0 +1,53 @@
+import valueParser from "postcss-value-parser"
+import {
+  cssWordIsVariable,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "function-url-data-uris"
+
+export const messages = ruleMessages(ruleName, {
+  expected: "Expected data uris in function url",
+  rejected: "Unexpected data uris in function url",
+})
+
+export default function (expectation) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, {
+      actual: expectation,
+      possible: [
+        "always",
+        "never",
+      ],
+    })
+    if (!validOptions) { return }
+
+    root.walkDecls(function (decl) {
+      valueParser(decl.value).walk(valueNode => {
```

Comment:
@davidtheclark Somebody can use the non-standard properties for urls (then use postcss plugins transform their to property with url args) (in sass sometimes i see variable `$background-image: url('foo.png')`), if we support this all ok, if not support i can made array with declaration only accepts url as argument.